plugin.yaml

```diff
@@ -879,6 +879,40 @@ node_types:
             openstack_config:
               default: {}
 
+  cloudify.openstack.nodes.Stack:
+    derived_from: cloudify.nodes.Root
+    properties:
+      template:
```

Comment:
change `template` to `files` which should all override by files parameter:
https://developer.openstack.org/api-ref/orchestration/v1/index.html#stacks.

Reply:
Can I rename parameter to "template_file"?
Because in openstack API parameters "files" used for supplies the contents of files referenced in the template. But we must specify template itself.